app/templates/_bower.json

```diff
@@ -36,24 +36,32 @@
         <%_ if (websocket == 'spring-websocket') { _%>
         "sockjs-client": "1.0.3",
         "stomp-websocket": "2.3.4",
+        <%_ } _%> <%_ if (enableSocialSignIn) { _%>
+        "bootstrap-social": "4.10.1",
         <%_ } _%>
+        "font-awesome": "4.4.0",
```

Comment:
Adding font-awesome is probably going to add a lot of resources, at the moment we only use "normal" bootstrap -> the idea is that we provide something "light" by default, but it's easy for people to extend (just do a Bower install). Can you remove this, and add the Bootstrap class instead?